beacon/beacon.go

```diff
@@ -69,8 +68,8 @@ type layerClock interface {
 	LayerToTime(types.LayerID) time.Time
 }
 
-// New returns a new TortoiseBeacon.
-func New(
+// NewProtocolDriver returns a new ProtocolDriver.
+func NewProtocolDriver(
```

Comment:
can we leave as New please? what the point of having verbose initializer if whole module is meant to provide a single initializer?

Reply:
done